docs/features/pm2-api.md

```diff
@@ -60,59 +58,57 @@ pm2.connect(function(err) {
 
 * `script` - The path of the script to run.
 * `jsonConfigFile` - The path to a JSON file that can contain the same options as the `options` parameter.
-* `errback(err,proc)` - An errback called when the `script` has been started. 
+* `errback(err,proc)` - An errback called when the `script` has been started. The `proc` parameter will be a [pm2 process object](https://github.com/soyuka/pm2-notify#templating).
```

Comment:
What about trying to document the internal `process` object on the same page? For example, you can link to the `describe` section below.